regression/cbmc-incr-smt2/pointers/pointer_offset.c

```diff
@@ -0,0 +1,16 @@
+int main()
+{
+  int a;
+  int *p = &a;
+  int *q = &a;
+
+  __CPROVER_assert(
+    __CPROVER_POINTER_OFFSET(p) != __CPROVER_POINTER_OFFSET(q),
+    "expected failure because offsets should be the same");
+
+  // TODO: Remove comments once pointer arithmetic works:
+
+  // *q = p + 2;
+
+  // __CPROVER_assert(__CPROVER_POINTER_OFFSET(p) != __CPROVER_POINTER_OFFSET(q), "expected failure");
```

Comment:
:pick: This assertion looks like it should pass once we have support for integer arithmetic. The assignment above will give `q` an offset of `2 * sizeof(int)` and p will still have an offset of zero.